packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/PieMarkerConfigurationMenu.tsx

```diff
@@ -30,6 +30,7 @@ interface MarkerConfiguration<T extends string> {
 
 export interface SharedMarkerConfigurations {
```

Comment:
I guess this could be moved to a shared.ts file?

Reply:
Yes, perhaps